examples/truffle/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "ethcontract-contracts",
   "version": "0.1.0",
-  "private": "true",
+  "private": true,
   "description": "Test contracts for ethcontract-rs runtime and proc macro.",
   "scripts": {
-    "develop": "truffle develop",
-    "build": "truffle build",
-    "start": "(echo 'migrate' && cat) | truffle develop",
-    "test": "truffle test"
+    "build": "rimraf build && truffle build && node scripts/networks.js inject",
```

Comment:
Why do we need this dependency? Can't you just run `rm -rf` directly from here?

Reply:
rimraf is cross-platform. I guess its not super important but it will work on Windows as well.